front/src/CommonStylesheets/form.scss

```diff
@@ -1,3 +1,7 @@
+$color-dark-brown: rgb(31, 31, 31);
```

Comment:
You can move all those color variables in separate file as now you have the same variables redefined in different places